core/baseline-core-v1.0-psd01.md

```diff
@@ -191,8 +191,273 @@ The ability of a Party operating Workflows on a baseline-compliant implementatio
+
+**[R12]**	A BPI SHOULD have the same liveness properties as the DLT used to implement it.
+
+Liveness means that if a DLT does not require (commercial) counterparties to constantly monitor its state to ensure that the state of the DLT is correct, then the BPI should not require a constant observation of its state either.
+
+**[R13]**	A BPI MUST be censorship resistant.
+
+Censorship resistant means that a (commercial) counterparty can terminate a commercial transaction at any time without another counterparty or any Node of the DLT used to implement the BPI being able to stop the termination of the commercial transaction.
+
+**[R14]**	A BPI MUST be able to provide privacy of the (commercial) conterparties data with respect to any party outside of the BPI.
+
+ 
+## 2.7 High-Level Functional Requirements
+
+This section describes the prerequisites and high-level general operational framework requirements 
+* Functional Requirements on commercial counterparties
+* DLT-based Lifecycle Processes
+
+### 2.7.1	Functional Requirements on commercial counterparties 
+
+This section states the commercial and operational functionalities required from commercial counterparties.
+
+**[R15]**	Commercial Counterparties MUST have the ability to meet all required legal, compliance and business reporting requirements. 
+
+This comprises, e.g., fraud or tax audit requirements based on commercial transactions on a BPI.
+
+**[R16]**	Commercial Counterparties MUST support the Reference Architecture defined in this section.
+
+**[R17]**	Commercial Counterparties MUST use the Baseline Protocol APIs to transact on a commercial state object.
+
+An ability of a Requester to request products, services or asset, in other words commercial state objects, through an instance of the Baseline Protocl APIs does not necessarily imply ability to provide products, services and assets through an instance of the Baseline Protocol APIs and vice versa.
+
+It is important for (commercial) counterparties to know what level of conformance other (commercial) counterparties have with the Baseline Protocl Standard.
+
+**[R18]**	Counterparties MUST publish their con-formance (self-declaration or certification) with the Baseline Protocol standard in a publicly accessible manner.
+
+### 2.7.2	DLT-based Lifecycle Processes
+
+Commercial Counterparties must comply efficiently and effectively with requirements of regulatory frameworks, e.g., Office of Foreign Assets Control ("OFAC") of the US Department of the Treasury) when employing new operational and commercial frameworks as laid out in this standard.
+
+**[R19]**	If required to meet particular third party requirements, (e.g., privacy or regu-latory frameworks in different jurisdictions), a commercial counterparty MUST record a pseudonymous map of the supply chain that is required to fulfill the provisioning of a requests commercial state object (products, services or assets) transacted on a BPI.
```

Comment:
why is regu-latory written in this way? I've seen this a few times in the document, so there must be a reason

Reply:
requests's

Reply:
State Object